plugins/statuslines/lightline.nix

```diff
@@ -72,6 +72,27 @@ in {
         });
       };
 
+      inactive = mkOption {
+        default = null;
+        type = types.nullOr (types.submodule {
+          options = let
+            listType = with types; nullOr (listOf (listOf str));
```

Comment:
listType = with helpers.nixvimTypes; maybeRaw (listOf (listOf str)).

This allows users to add lua code everywhere.